### PR TITLE
WMCO 10.15.1 release notes

### PR DESCRIPTION
**Pull request by mburke5678**
WMCO 10.15.0 needed to be dropped. WMCO 10.15.1 replaces it.

Errata:

Preview:
https://72190--ocpdocs-pr.netlify.app/openshift-enterprise/latest/windows_containers/windows-containers-release-notes-10-15-x#wmco-10-15-1

### CODE REVIEW

File: windows_containers/windows-containers-release-notes-10-15-x.adoc [hunk 1]
```diff
@@ -28,36 +28,39 @@ For more information, see the Red{nbsp}Hat OpenShift Container Platform Life Cyc
...
 
-=== New features and improvements
-[id="wmco-10-15-0-node-certificates"]
 
-[id="wmco-10-15-0-numbering"]
-==== New WMCO numbering
+[id="wmco-10-15-1"]
+== Release notes for Red{nbsp}Hat Windows Machine Config Operator 10.15.1
+
+This release of the WMCO provides new features and bug fixes for running Windows compute nodes in an {product-title} cluster. The components of the WMCO 10.15.1 were released in link:https://access.redhat.com/errata/RHBA-2024:1191[RHBA-2024:1191].
```
Comment (cbippley) [inline windows_containers/windows-containers-release-notes-10-15-x.adoc:40]:
```suggestion
This release of the WMCO provides new features and bug fixes for running Windows compute nodes in an {product-title} cluster. The components of the WMCO 10.15.1 were released in link:https://access.redhat.com/errata/RHBA-2024:1191[RHBA-2024:1191].
```
I assume the link will work once the errata is shipped live, yes?

**Review by cbippley — COMMENTED** [review]
Other than the RHBA link not currently working, LGTM!

File: windows_containers/windows-containers-release-notes-10-15-x.adoc (hunk 1 above)
Comment (mburke5678) [reply windows_containers/windows-containers-release-notes-10-15-x.adoc:40]: Yes. Sorry. I should have mentioned that.

**Pull request merged**